### PR TITLE
🏃 refactor code to replace bmm by m3m

### PR DESCRIPTION
**Pull request by maelk**
**What this PR does / why we need it**:
This PR does a bit of code refactoring

### CODE REVIEW

File: baremetal/metal3machine_manager_test.go [hunk 1]
```diff
@@ -78,26 +78,26 @@ func bmmSpecAll() *capm3.Metal3MachineSpec {
...
 		},
 	}
 }
 
 func consumerRef() *corev1.ObjectReference {
 	return &corev1.ObjectReference{
-		Name:       "mybmmachine",
+		Name:       "mym3machine",
 		Namespace:  "myns",
 		Kind:       "BMMachine",
```
Comment (fmuyassarov) [inline baremetal/metal3machine_manager_test.go:102]: Should the value of ` Kind:` be kept as is?

Comment (jan-est) [reply baremetal/metal3machine_manager_test.go:102]: Should it be M3Machine?

Comment (maelk) [reply baremetal/metal3machine_manager_test.go]: thanks, I replaced only bmm, so I updated the patch to change BMM also

**Comment by maelk** [comment]
Another XL PR, but it is only a search and replace, so should be quite straightforward to review. Sorry for that

**Comment by furkatgofurov7** [comment]
lgtm

**Pull request merged**